test/mocks/urlfetchapp.js

```diff
@@ -0,0 +1,28 @@
+var Future = require('fibers/future');
+
+var MockUrlFetchApp = function() {
+  this.delayFunction = () => 0;
+  this.resultFunction = () => '';
+};
+
+MockUrlFetchApp.prototype.fetch = function(url, optOptions) {
+  var delay = this.delayFunction();
+  var result = this.resultFunction();
+  if (delay) {
+    sleep(delay).wait();
+  }
+  return {
+    getContentText: () => result,
+    getResponseCode: () => 200
+  };
+};
+
+function sleep(ms) {
+  var future = new Future;
```

Comment:
I can't find the exact rule, but it's common to include parenthesis for constructors.
```js
var future = new Future();
```
li 2 implies parens after constructors: https://google.github.io/styleguide/jsguide.html#formatting-where-to-break

Reply:
Done